cmd/kubeadm/app/cmd/token_cmd_test.go

```diff
@@ -0,0 +1,67 @@
+// +build cmd
```

Comment:
This is all really cool! Really glad to see this and more testing going into place.  I would just suggest that we shouldn't have a `token_cmd_test.go` file, since that doesn't follow the Go standard for naming test files.  I think everything inside of `token_cmd_test.go` should just get placed in `token_test.go`.